### DISCUSSION TITLE
Is there a known issue of .click({force: true}) not working properly?

### DISCUSSION THREAD

**Discussion by quad5** [cypress-io/cypress#9535] [Questions and Help]
I have a same test where I specify `{force: true}` and `{force: false}` in different run, and the result is different in the sense that it seems like element wasn't clicked.

For sure, element is visible.

**Comment by dmtrKovalenko** [comment]
Force true is not doing any visibility checks, so its directly firing the event on the target. It means that event wasn’t fired for you — the code itself did not handle the event.
